src/ui/MessageInput/index.jsx

```diff
@@ -122,14 +124,14 @@ const MessageInput = React.forwardRef((props, ref) => {
   useEffect(() => {
     if (!isEdit) {
       setIsInput(false);
-      document.getElementById(TEXT_FIELD_ID).innerHTML = '';
+      document.getElementById(textFieldId).innerHTML = '';
```

Comment:
use ref for finding the DOM element as we discuss in https://github.com/sendbird/sendbird-uikit-react/pull/361